shared/macros/10-oval.jinja

```diff
@@ -84,9 +91,19 @@ Generates the :code:`<affected>` tag for OVAL check using correct product platfo
     </criteria>
     {{%- endif %}}
   </definition>
+
+  {{% if xccdf_variable %}}
+  {{{ oval_line_in_file_define_variable(xccdf_variable, datatype="string") }}}
```

Comment:
I don't like that  this macro generates a local variable that references an external variable. Also, I don't like that the generated local variable has a comment: `Regex that matches exact value represented by var_selinux_policy_name XCCDF variable`. That is misleading, at least for the OVAL in rule `selinux_policytype`, the XCCDF value isn't a regular expression and the local variable isn't used in a pattern match operation but in a equals operation.

Reply:
I hope I resolved naming problems in 68a527c954b4f381a9033f146d6a475e45296c9d  .
Regarding the usage of local variable... I always thought that you need a local variable to reference external variable and then the local variable can be used in OVAL state. Am I wrong?

Reply:
You can use external variable directly in OVAL state. You don't need the interim local variable. See for example `linux_os/guide/system/selinux/selinux_state/oval/shared.xml` where in `textfilecontent54_state` the `var_ref="var_selinux_state"` references `external_variable`.